lib/auth/github.go

```diff
@@ -65,6 +69,9 @@ func (a *Server) CreateGithubAuthRequest(ctx context.Context, req types.GithubAu
 
 // upsertGithubConnector creates or updates a Github connector.
 func (a *Server) upsertGithubConnector(ctx context.Context, connector types.GithubConnector) error {
+	if err := checkGithubFeatureSupport(connector, nil); err != nil {
```

Comment:
Pass `ctx` to this function.

Reply:
I'm actually thinking, we should probably remove this check from connector creation also. This will slow it down and make it impossible to create the connector if Github is having issues for example.

Reply:
Talked to @jakule about it, we decided that it's fine to leave check in when creating auth connectors